README.md

```diff
@@ -167,6 +167,19 @@ and run tests with
 
 For more information on the integration tests, run `test_integration --help`.
 
+## Building the templates
+
+1. Create a fedora-28 AppVM for building
+2. Increase the disk size to at least 15GB (as the build uses over 10GB)
```

Comment:
I've been struggling to resize my root volume; even when it works, and dom0 does not emit a cryptic error, I still run out of disk space. What I've been doing is resizing the _private_ volume for the build VM, which is both more reliable in terms of the resizing operation's output, and properly accommodates the filesystem-hungry build process.

Noting this rather than requesting changes because it seems more research required; we'll finetune the docs as we use these features more regularly during template maintenance.